Sources/Responses/responseGenerator.swift

```diff
@@ -48,8 +48,16 @@ public class ResponseGenerator {
     public func generate404Response() -> HttpResponse {
         return HttpResponse(version: "HTTP/1.1",
                             statusCode: 404,
-                            statusPhrase: "NotFound",
+                            statusPhrase: "Not Found",
```

Comment:
Is there a way that you could automatically set the phrase if you set the code (or vice versa)? Setting the phrase and code separately might lead to some subtle errors. What if you made a spelling mistake?